src/main/java/mate/controller/AddDriverToCarController.java

```diff
@@ -0,0 +1,37 @@
+package mate.controller;
+
+import java.io.IOException;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import mate.lib.Injector;
+import mate.model.Car;
+import mate.model.Driver;
+import mate.service.CarService;
+import mate.service.DriverService;
+
+public class AddDriverToCarController extends HttpServlet {
+    private static final Injector injector = Injector.getInstance("mate");
+    private final DriverService driverService = (DriverService) injector
+            .getInstance(DriverService.class);
+    private final CarService carService = (CarService) injector
+            .getInstance(CarService.class);
+
+    @Override
+    protected void doGet(HttpServletRequest req, HttpServletResponse resp)
+            throws ServletException, IOException {
+        req.getRequestDispatcher("/WEB-INF/views/car/driver/add.jsp").forward(req, resp);
+    }
+
+    @Override
+    protected void doPost(HttpServletRequest req, HttpServletResponse resp)
+            throws ServletException, IOException {
+        Long driverId = Long.valueOf(req.getParameter("driver_id"));
+        Driver driver = driverService.get(driverId);
+        String carId = req.getParameter("car_id");
+        Car car = carService.get(Long.valueOf(carId));
```

Comment:
try to follow one code style in you app:
create the same type for the similar values everywhere
```suggestion
        Long driverId = Long.valueOf(req.getParameter("driver_id"));
        Driver driver = driverService.get(driverId);
        Long carId = Long.valueOf(req.getParameter("car_id"));
        Car car = carService.get(carId);
```
or
```suggestion
        String driverId = req.getParameter("driver_id");
        Driver driver = driverService.get(Long.valueOf(driverId));
        String carId = req.getParameter("car_id");
        Car car = carService.get(Long.valueOf(carId));
```